state-chain/chains/src/dot.rs

```diff
@@ -996,7 +996,7 @@ impl BenchmarkValueExtended for PolkadotChannelId {
 	}
 }
 
-#[cfg(debug_assertions)]
```

Comment:
I think just in the const's case, it should be `cfg(test)`.

Reply:
This then solves the problem, and tests can be run in release or debug.

Reply:
True